lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1073,6 +1073,12 @@ def cla(self):
             self.zaxis._set_scale(self._sharez.zaxis.get_scale())
         else:
             self.zaxis._set_scale('linear')
+            try:
+                self.set_zlim(0, 1)
+            except TypeError:
+                pass
+            except AttributeError:
```

Comment:
It's line 737 of mplot3d/axes3d.py:
```
E           AttributeError: 'Axes3DSubplot' object has no attribute 'callbacks'
 
mplot3d/axes3d.py:737: AttributeError
```

You can find the longer version [here](https://pastebin.com/jHANenSQ)

thanks!